### PR TITLE
Episode 1

### PR DESCRIPTION
**Pull request by bobturneruk**
I hope this addresses the notes in #25 around episode 1, specifically. I think it's worth getting this reviewed before looking at another episode!

I feel like there is more to be done on this, but I reckon it will flow better when delivered, even if some of the text could still be cut and refined.

### CODE REVIEW

File: _episodes/01-getting-started-with-conda.md [hunk 1]
```diff
@@ -198,7 +202,8 @@ your usual version of Python in your normal environment.
 
 ## Why use Conda?
 
-Whilst there are many different package and environment management systems that solve either the
+In Python there is a built in package manager `pip` and virtual environment manager `venv`, so
+why use Conda? Whilst there are many different package and environment management systems that solve either the
 package management problem or the environment management problem, Conda solves both of these
 problems and explicitly targeted at (data) science use cases.
```
Comment (ns-rse) [inline _episodes/01-getting-started-with-conda.md]: Not your edit but needs an `is` in here or changing `targeted` to `targets`...

```suggestion
problems and is explicitly targeted at (data) science use cases.
```

...org...

```suggestion
problems and explicitly targets (data) science use cases.
```

File: _episodes/03-using-packages-and-channels.md [hunk 2]
```diff
@@ -15,27 +14,6 @@ keypoints:
 - "Understand how to use Conda and Pip together effectively."
 ---
 
-## What are Conda packages?
```
Comment (ns-rse) [inline _episodes/03-using-packages-and-channels.md:18]: Perhaps we should leave a link here to say something along the lines of...

```
We have covered the anatomy of Conda Packages in [Section 1](01-getting-started-with-conda).
```
?

File: _extras/guide.md [hunk 3]
```diff
@@ -2,7 +2,31 @@
...
+Library/Package are often used interchangeably.
+
+# Environment management systems for Python
+
+Conda is not the only way; Python for example has many more ways of working with environments:
+* [virtualenv](https://virtualenv.pypa.io/en/latest/)
+* [pipenv](https://pipenv.pypa.io/en/latest/)
+* [venv](https://docs.python.org/3/library/venv.html)
+* [pyenv](https://github.com/pyenv/pyenv)
+* ...
```
Comment (ns-rse) [inline _extras/guide.md]:
```suggestion
* [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/)
```

**Review by ns-rse — APPROVED** [review]
Some minor comments but I think these are great improvements and makes it flow better.

File: _extras/guide.md (hunk 3 above)
Comment (bobturneruk) [reply _extras/guide.md]: Nice.

File: _episodes/03-using-packages-and-channels.md (hunk 2 above)
Comment (bobturneruk) [reply _episodes/03-using-packages-and-channels.md:18]: Hopefully I've dealt with this by imporving the title of episode 3, which is now about how to use channels and work with Conda and PyPI. We should look at ep3 again soon.

**Pull request merged**